src/app/seeds/development/challenges.json

```diff
@@ -1,16 +1,161 @@
 {
   "challenges": [
     {
-      "id": "6139408e85e723a0a87fdbc7",
+      "id": "613a2d2b3084347fbc8efc03",
```

Comment:
You are already using this ID for a Challenge Platform. Technically it's not a problem but this can lead to confusion.